services/viva/microservice/src/lambdas/syncWorkflow.ts

```diff
@@ -104,22 +101,28 @@ function updateCaseDetailsWorkflow(keys: CaseKeys, newWorkflow: VivaWorkflow) {
   return dynamoDb.call('update', updateParams);
 }
 
-export async function syncWorkflow(input: LambdaRequest, dependencies: Dependencies) {
+function isSetWorkflowId(caseItem: CaseItem) {
```

Comment:
Do we want to specify the return value even in these cases? Just asking. The return values for the other functions in this file are overall inconsistent.